app.css

```diff
@@ -36,3 +36,37 @@ body {
 .head-wrapper header ul li:hover {
     border-bottom: 1px solid #4cc5e2;
 }
+
+/*Slick*/
+.single-item {
+  direction: ltr;
+  margin: auto;
+  margin-top:80px;
```

Comment:
Instead of these 2 rules you can just do:
`margin: 80px auto auto;`
However I don't think margin bottom should be set to `auto`, but to zero.